### DISCUSSION TITLE
GEOL_C or CONJ

### DISCUSSION THREAD

**Discussion by richard-sisson** [australiangeomechanics/agsformat#48] [Q&A] [answered]
Hi Phil,

The GEOL_C list is referenced in 4.1.1AU1.2 by two headings storing joiner values (i.e. GEOL_1SHC and GEOL_2SHC)

Would the CONJ list be more appropriate?

Thanks
Richard

**Comment by SolidRockProgrammer** [comment, marked as answer]
Hi Richard
We neglected to document how to use GEOL_1C and GEOL_2C, and their ABBR list GEOL_C.
GEOL_1C and GEOL_2C are X type, and any combination of Codes from the GEOL_C ABBR list may be transmitted in this heading.  

This concept is not natively in AGS 4.1.  Now I think about it, I should propose the AGS UK that a new data type is added to support this.

regards

Phil

**Reply by richard-sisson** [reply]
Hi Phil,

Sorry, I was talking about the Shape Joiner headings:
![image](https://user-images.githubusercontent.com/10456433/186576801-0bfc2b97-a9f8-42f5-88bf-2e064201fdb6.png)

Thanks
Richard

**Reply by SolidRockProgrammer** [reply]
Yes, we should have set GEOL_1SHC and GEOL_2SHC values for ABBR_HDNG to: CONJ
That is now applied to draft v1.3

**Reply by richard-sisson** [reply]
Cheers Phil
